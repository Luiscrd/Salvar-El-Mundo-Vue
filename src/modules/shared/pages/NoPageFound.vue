<template>
    <div class="box-center d-flex align-items-center">
        <div class="container mt-5">
            <h1>Page not foud</h1>
            <img src="@/assets/404.png" alt="404">
            <div>
                <button type="button" class="btn btn-primary">Volver</button>
            </div>
        </div>
    </div>

</template>

<script>
export default {
    name: '404',
    setup() {
    }
}
</script>

<style lang="scss" scoped>
.box-center {
    background-image: url('@/assets/apocalipsis_fin_del_mundo_profecias.jpg_1834093470.png');
    background-repeat: no-repeat;
    background-size: cover;
    height: calc(100vh - 160px);
    text-align: center;
    color: white;
}
</style>