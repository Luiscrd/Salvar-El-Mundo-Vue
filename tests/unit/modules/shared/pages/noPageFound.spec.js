import { shallowMount } from '@vue/test-utils'
import NoPageFound from '@/modules/shared/pages/NoPageFound.vue'

describe('Formulario View', () => {

    let wrapper;

    beforeEach(() => {
          
          wrapper = shallowMount(NoPageFound)

    });

    test('Debe de hacer mach con el snapshot', () => {

        expect(wrapper.html()).toMatchSnapshot()

    })

    test('El texto a mostrar en el h1 debe ser "Page not foud"', () => {

        const h1 = wrapper.find('h1')

        expect(h1.text()).toBe('Page not foud')

    })

})