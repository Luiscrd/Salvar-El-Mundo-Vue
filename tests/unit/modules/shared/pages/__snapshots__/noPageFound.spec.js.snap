// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Formulario View Debe de hacer mach con el snapshot 1`] = `
<div class="box-center d-flex align-items-center">
  <div class="container mt-5">
    <h1>Page not foud</h1><img src="" alt="404">
    <div><button type="button" class="btn btn-primary">Volver</button></div>
  </div>
</div>
`;
